src/components/ModalPortal.js

```diff
@@ -207,7 +207,7 @@ export default class ModalPortal extends Component {
     }
 
     if (this.props.shouldCloseOnEsc && event.keyCode === ESC_KEY) {
-      event.preventDefault();
```

Comment:
Does Remove prevenDefault break something? Why was there before?

Reply:
There is no need to `preventDefault` in this case, but instead to let only modal that receives this event to process it.